doc/Language/unicode_entry.pod6

```diff
@@ -103,6 +103,12 @@ Further information about entering special characters in Vim can be found on
 the Vim Wikia page about L<entering special
 characters|http://vim.wikia.com/wiki/Entering_special_characters>.
 
+=head3 vim-perl6
+
+The L<https://github.com/vim-perl/vim-perl6 | vim-perl6> plugin for Vim can be
+configured to replace ASCII based ops to their Unicode based equivalent. This
+will convert the ASCII based ops on the fly while typing them.
```

Comment:
It might be worth mentioning that this is just `abbrev` expansion?

Reply:
This is explained on the README of the plugin (if https://github.com/vim-perl/vim-perl6/pull/17 gets accepted). I could add it here too, but I thought implementation specific details should go in the implementation's own docs.

Reply:
Fair point.

Perhaps add in 'optionally' ('will optionally convert') since `abbrev`s are user configurable?